lib/multi_mail/mailgun/message.rb

```diff
@@ -8,66 +8,75 @@ class Mailgun < MultiMail::Message::Base
       #
       # @return [Multimap] the message headers in Mailgun format
       def mailgun_headers
-        hash = Multimap.new
+        mm = Multimap.new
         header_fields.each do |field|
           key = field.name.downcase
-          unless %w(from to cc bcc subject tag).include?(key)
-            hash["h:#{field.name}"] = field.value
+          unless %w(from to cc bcc subject tag reply-to).include?(key)
+            mm["h:#{field.name}"] = field.value
           end
         end
-        hash
+        mm.instance_variable_get('@hash')['h:Reply-To'] = self['Reply-To'].value if self['Reply-To']
```

Comment:
Why is it necessary to use `instance_variable_get` instead of `mm['h:Reply-To'] = ...`?